### ISSUE TITLE
Instagram login error: oops an error occurred

### ISSUE THREAD

**Issue by ghost** [InstaPy/InstaPy#2628]
Hi guys,

since yesterday when i try to login to instagram , after username and password form submitted it answer with :

oops an error occurred

and i can't do anything ...

any idea @timgrossmann @sionking @uluQulu ?

Thanks

![error](https://user-images.githubusercontent.com/1229160/43689028-08b9a5f8-98f4-11e8-93c2-25982213a81f.png)

**Comment by Areahints** [comment]
Hi @ilmetu I think there's an API change regarding cookie and crsftoken which is creating troubles with Instapy and other automation software, at first I thought it was based on IP, but its been reported across instabot and the other ones I use, basically if you haven't cleared your cookies you won't notice a thing, its when the software has to create a new cookie that all sort of surprises spring up, yesterday 2 account that had been running fine failed to login, **and one only worked when I used email verification**, so far the only solution that has been offered is this  (taken from another issue of an automation bot)

```
100% WORKING SOLUTION!:

- Temporary deactivate account
- Select reason: just need a break
- Wait 2-3 hours
- Login directly with bot (It will activate account back without sentry_block or checkpoint_challenge_required errors)
```
im skeptical about trying this cos of the difference in my ip locations (2 different continents!) but ive temporarily disabled all accounts on instapy and im waiting to see who else has a solution

**Comment by ghost** [comment]
Thanks @Areahints i hope someone else find a different solution...
but thanks a lot for this info.

i'm not alone and this makes me feel better :+1:

**Comment by Areahints** [comment]
hi @ilmetu i was able to logon all previous accounts i had, but i couldnt add new IG accounts, hope your end is fixed?

**Comment by ghost** [comment]
At the moment the only way i found to solve the problem is by using a strong anonimization with a residential proxy provided by [luminati](https://luminati.io/?affiliate=ref_5b43dfdfeebad2ae1d7ff104)

login with them solve all of my issues.

hope it helps for you too.

**Comment by Areahints** [comment]
im signing up on luminati rn, can you show me how you use proxy in quickstart.py, im going for the 7 day trial since they offer proxies in my country

**Comment by ghost** [comment]
first of all you can try with the [luminati](https://luminati.io/?affiliate=ref_5b43dfdfeebad2ae1d7ff104) chrome extension to login with your computer, maybe using a "incognito" chrome tab and then you can try to delete the cookies and login again with instapy

**Comment by Areahints** [comment]
i use instapy on linux server via ssh and i've never seen the chrome browser activate or run :(

**Comment by rohanvivek** [comment]
https://github.com/timgrossmann/InstaPy/issues/2628#issuecomment-410542544
@Areahints regarding the same issue I'm facing the same issue my account is also temporary blocked in your given solution no page is opening in Instagram so how to disable the account in that case. Is there any alternative solution??

**Comment by henrich01** [comment]
Hello,

I had a copyright issue with Instagram and now when I click I acknowledge and proceed I can't login in my account but an error appears "Oops an error occurred". It has been more than 2 days now and my page is active and online but I can't find anyway on how to login. Can someone please help me?

**Comment by FezFamiliar** [comment]
@henrich01 , i have the same issue. I had a copyright issue and after i click ok, i get oops an error occured..

**Comment by kullshaan** [comment]
@henrich01 @FezFamiliar Same issue from past 1 day, copyright and acknowledge leads to opps an error occurred, in future if you get any changes like account back , please update here, i do not know how much time they will take my account has 37k followers and engagement was also awesome, feeling so down now.

**Comment by rupamsaha3564** [comment]
i have 10k followers on my instagram friends you can also search my account-rupam_saha3564
but when i am logging in its showing oops an error occured please help anyone it was really tough for me to reach that much

**Comment by nadia-am** [comment]
@rupamsaha3564 i have the same issue and i dont know what to do, if you find the solution please share it with me

**Comment by rupamsaha3564** [comment]
@nadia-am sure but friend are you on watsapp?and whats your instagram id?

**Comment by rupamsaha3564** [comment]
@nadia-am sure but friend are you on watsapp?and whats your instagram id?

**Comment by nadia-am** [comment]
@rupamsaha3564 actually i cant log in to my instagram account right now :(

**Comment by uzimwb** [comment]
[SOLVED]
i was facing the same problem as " oops a error occurred" when i logged in to my Instagram account i tried resting the password, uninstalling and reinstalling the app but i faced the same problem i was searching to have a contact with the support team , i emailed to "<info++aazq7ebr6xnkps@support.facebook.com  "  i got the response they replied that:
*Hi,*
*Thanks for contacting us. Before we can help, we need you to confirm that you own this account.*
*Please reply to this email and attach a photo of yourself holding a hand-written copy of the code below.*
*(secret code)*
*Please make sure that the photo you send:*
*- Includes the above code hand-written on a clean sheet of paper, followed by your full name and username*
*- Includes both your hand that's holding the sheet of paper and your entire face*
*- Is well-lit, and is not too small, dark or blurry*
*- Is attached to your reply as a JPEG file*
*Keep in mind that even if this account doesn't include any pictures of you or is used to represent someone or something else, we won't be able to help until we receive a photo that meets these requirements.*
*Thanks,*
*The Instagram Team*

**I sent them the required information and it resolved my problem within 8-12 hours.. i got y account back yupieee :D**

**Comment by uzimwb** [comment]
> @rupamsaha3564 [SOLVED]
i was facing the same problem as " oops a error occurred" when i logged in to my Instagram account i tried resting the password, uninstalling and reinstalling the app but i faced the same problem i was searching to have a contact with the support team , i emailed to "<info++aazq7ebr6xnkps@support.facebook.com  "  i got the response they replied that:
*Hi,*
*Thanks for contacting us. Before we can help, we need you to confirm that you own this account.*
*Please reply to this email and attach a photo of yourself holding a hand-written copy of the code below.*
*(secret code)*
*Please make sure that the photo you send:*
*- Includes the above code hand-written on a clean sheet of paper, followed by your full name and username*
*- Includes both your hand that's holding the sheet of paper and your entire face*
*- Is well-lit, and is not too small, dark or blurry*
*- Is attached to your reply as a JPEG file*
*Keep in mind that even if this account doesn't include any pictures of you or is used to represent someone or something else, we won't be able to help until we receive a photo that meets these requirements.*
*Thanks,*
*The Instagram Team*

**I sent them the required information and it resolved my problem within 8-12 hours.. i got y account back yupieee :D**

**Comment by uzimwb** [comment]
> @rupamsaha3564  Any problem regarding instagram 
mail me : uzairrafiqmw@gmail.com

**Comment by nadia-am** [comment]
@uzimwb i report the problem to instagram and they fixed it.

**Comment by uzimwb** [comment]
> @nadia-am same here they fixed it as i mentioned above
